openhtf/output/callbacks/__init__.py

```diff
@@ -22,13 +22,14 @@
 
 import base64
 import contextlib
-import cPickle as pickle
+import pickle as pickle
```

Comment:
This may cause a large slowdown on Python2.

Maybe
```
try:
   import cPickle as pickle
except:
   import pickle
```
from https://pymotw.com/2/pickle/
?